pkg/sql/opt/optbuilder/testdata/fk-on-delete-cascade

```diff
@@ -65,6 +65,42 @@ root
                 └── filters
                      └── child.p:9 = p:11
 
+# Delete with subquery; no fast path.
+build-cascades
+DELETE FROM parent WHERE EXISTS (SELECT p FROM parent)
```

Comment:
Oddly, this test doesn't error before the fix. I couldn't really figure out why exactly.